apps/runtime-demo/3005-runtime-host/src/App.tsx

```diff
@@ -18,11 +19,15 @@ const App = () => (
       <li>
         <Link to={'/remote2'}>remote2</Link>
       </li>
+      <li>
+        <Link to={'/remote3'}>remote3</Link>
+      </li>
     </ul>
     <Routes>
       <Route path="/" element={<Root />} />
       <Route path="/remote1" element={<Remote1 />} />
       <Route path="/remote2" element={<Remote2 />} />
+      <Route path="/remote3" element={<Remote3 />} />
```

Comment:
Behavior is odd... Loading the page directly attaches the CSS correctly, navigating to it through a link doesn't because css assets is missing in the manifest... It's almost like the built code caches something different than the hardened runtime which uses mf-manifest.json?